### ISSUE TITLE
a note on intellij-toml installation

### ISSUE THREAD

**Issue by xftroxgpx** [intellij-rust/intellij-rust#2212]
Following the instructions from https://github.com/intellij-rust/intellij-rust/tree/master/intellij-toml#installation

For those that don't use a DNS server(so, just me then?), in order to be able to see a non-empty list of plugins (and thus the Toml plugin entry too) in `Settings > Plugins > Browse repositories` you need hostname `plugins.jetbrains.com` (currently resolves to 52.208.50.140)

And to be able to install (eg. download) the `Toml` plugin, you need hostname `d9ernfkkvx1zk.cloudfront.net`.

Basically, `/etc/hosts`, for me, has:
```
52.208.50.140 plugins.jetbrains.com
54.192.2.204 d9ernfkkvx1zk.cloudfront.net
54.192.2.43 d9ernfkkvx1zk.cloudfront.net
54.192.2.239 d9ernfkkvx1zk.cloudfront.net
54.192.2.45 d9ernfkkvx1zk.cloudfront.net
```
